pkg/render/common/meta/meta.go

```diff
@@ -67,6 +68,26 @@ var (
 			Operator: corev1.TolerationOpExists,
 		},
 	}
+
+	// TolerateBootstrap tolerates the set of conditions that exist on a brand new cluster,
+	// useful to run components required to bootstrap networking.
+	TolerateBootstrap = []corev1.Toleration{
+		{
+			// Tolerate not-ready, which is the default toleration on new nodes.
+			// Set a TolerationSeconds so that we'll get rescheduled if a node goes NotReady
+			// in the future for an extended period of time.
+			Effect:            corev1.TaintEffectNoSchedule,
+			Key:               "node.kubernetes.io/not-ready",
+			TolerationSeconds: ptr.Int64ToPtr(300),
+		},
+		{
+			// Tolerate a node becoming unreachable by the apiserver for
+			// a short while.
+			Effect:            corev1.TaintEffectNoExecute,
+			Key:               "node.kubernetes.io/unreachable",
```

Comment:
Is there documentation about the taints that a node will get during bootstrap?

Reply:
Not specifically in the context of bootstrap, but it is discussed a bit here: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-based-evictions

While I do think it is probably a better experience for most users to be able to configure eviction properly, there is a risk that changing these will have unintended consequences for other users.

Reply:
What is the reasoning that bootstrap needs to tolerate "unreachable".
It doesn't seem like we would want to tolerate unreachable during bootstrap, because if a node was unreachable then we wouldn't want typha to be scheduled to that node anyway.